eofconvention.fsx

```diff
@@ -0,0 +1,29 @@
+open System.IO
+
+let EOFIsEOL(path: FileInfo) = 
+    try
+        use sr = new StreamReader (path.FullName)
```

Comment:
@realmarv sr is not in our list of blessed abbreviations :-P you can use `streamReader` here, unless there's an ogre sitting next to you charging you 1 penny for every key stroke

Reply:
Done